privacyidea/lib/tokenclass.py

```diff
@@ -270,6 +270,26 @@ def add_tokengroup(self, tokengroup=None, tokengroup_id=None):
                             tokengroupname=tokengroup).save()
         return r > 0
 
+    @property
+    def owners(self):
+        """
+        return all the owners of a token
+        If the token has no owner assigned, we return an empty list
+
+        :return: The owners of the token
+        :rtype: A list of user object
+        """
+        user_objects = []
+        tokenowners = self.token.all_owners
+        if tokenowners:
+            for tokenowner in tokenowners:
+                username = get_username(tokenowner.user_id, tokenowner.resolver)
+                user_object = User(login=username,
```

Comment:
As far as I remember, the problem is, that the user object will not contain the login name, since the constructor acutally does not check for the user in the resolver.

Reply:
OK, thanks. You are right.
Strangely at this point initially I ended up with an incomplete user object without a login name to dump.
I am not sure what happened back then. Checked again and changed it in df240c350